### DISCUSSION TITLE
How to make the rename select the name part when you start the rename?

### DISCUSSION THREAD

**Discussion by OriLiMu** [sxyazi/yazi#1208] [1. Q&A] [answered]
I want the name of the file is selected when I start rename. Then I can directly input the name I want to change, not delete the whole name first. 
How to make it?

**Comment by lpnh** [comment, marked as answer]
I use this: https://github.com/sxyazi/yazi/discussions/1079#discussioncomment-9531593

My `keymap.toml`:

```
[[manager.prepend_keymap]]
on = ["R"]
run = "rename --empty=stem --cursor=start"
desc = "Easy rename"
```
